spec/i18n_spec.rb

```diff
@@ -281,7 +281,7 @@ def allowed_untranslated_key?(locale, key)
     )
   end
 
-  it 'has matching HTML tags' do
+  it 'has matching HTML tags across all locales' do
```

Comment:
Author's note: Updated the language of this test to be more clear as to what it was actually testing. At first, I thought it was mad that the HTML tags weren't balanced, but it was actually checking to ensure that the same tags are used across all locales.